tests/pcmsolver/ccsd-pte/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+include(TestingMacros)
+
+add_regression_test(pcmsolver-ccsd-pte "psi;quicktests;pcmsolver;addon;ccsd;pte")
```

Comment:
no, they don't hurt. except for quick and smoke, nobody uses the labels except maybe developers for their own convenience.